src/_restore.sh

```diff
@@ -76,8 +76,10 @@ function restore_doPostgresRestore() {
       log "Extracting backup for $db"
       debug "`compose_client exec -T $postgresComposeService\
         tar -xvzf /backups/$backupFile ./$db.psql 2>&1`"
-      log "Restoring $db"
-      dbRestoreFlags="-d $db --clean --if-exists --no-owner --disable-triggers --verbose"
+      dbAdminEnvvar="PG_${db^^}_ADMIN_USER"
```

Comment:
Ahh yes, `^^`. Completely intuitive way to convert to upper case 🤣 . Oh bash, never change.